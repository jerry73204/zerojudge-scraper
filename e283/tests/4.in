20
1 0 0 0
0 1 1 1
0 1 0 1
1 1 0 1
0 0 1 0
0 1 1 1
0 1 1 1
0 1 0 1
0 1 0 1
0 1 1 1
1 1 0 0
0 1 0 1
0 1 1 1
1 0 0 0
0 1 0 1
1 1 0 0
0 0 1 0
1 1 0 1
1 0 0 0
0 0 1 0
1
1 0 0 0
1
0 0 1 0
14
0 1 1 1
0 1 0 1
1 1 0 1
0 1 0 1
0 1 0 1
1 1 0 0
0 1 0 1
1 0 0 0
0 1 0 1
1 1 0 1
0 0 1 0
1 1 0 1
1 0 0 0
0 1 0 1
11
0 1 1 1
1 0 0 0
1 1 0 1
1 1 0 0
0 0 1 0
1 0 0 0
1 1 0 1
0 1 1 1
0 0 1 0
1 0 0 0
1 1 0 0
11
0 1 0 1
1 0 0 0
1 1 0 1
0 0 1 0
1 1 0 1
0 1 0 1
0 0 1 0
1 1 0 0
0 1 1 1
0 1 1 1
1 0 0 0
19
0 1 0 1
0 1 0 1
1 0 0 0
1 1 0 0
1 0 0 0
0 1 0 1
1 0 0 0
0 1 0 1
0 0 1 0
0 1 0 1
1 1 0 1
0 0 1 0
1 1 0 0
0 0 1 0
0 1 1 1
0 0 1 0
1 1 0 1
1 1 0 0
0 1 0 1
11
0 1 0 1
1 1 0 0
0 1 0 1
0 0 1 0
1 1 0 0
0 1 1 1
0 1 0 1
1 0 0 0
0 1 1 1
1 1 0 0
1 1 0 0
8
0 1 0 1
0 1 0 1
1 1 0 1
0 1 0 1
0 1 0 1
1 0 0 0
0 1 0 1
0 1 0 1
10
1 1 0 1
1 0 0 0
1 1 0 1
0 0 1 0
0 0 1 0
0 1 1 1
1 0 0 0
1 0 0 0
0 1 1 1
0 1 1 1
15
0 0 1 0
1 0 0 0
1 1 0 1
1 0 0 0
0 1 1 1
1 1 0 1
1 1 0 0
0 1 1 1
1 1 0 1
0 0 1 0
1 0 0 0
1 0 0 0
1 1 0 0
1 1 0 1
0 1 1 1
14
0 1 0 1
0 1 1 1
1 0 0 0
0 1 1 1
1 1 0 0
1 0 0 0
0 1 0 1
1 0 0 0
0 1 0 1
1 0 0 0
1 0 0 0
1 1 0 1
1 1 0 1
1 0 0 0
1
1 1 0 1
2
1 1 0 0
1 0 0 0
19
1 1 0 1
0 1 0 1
0 0 1 0
0 1 0 1
1 0 0 0
1 1 0 0
0 0 1 0
0 1 1 1
0 0 1 0
1 0 0 0
0 1 1 1
1 1 0 1
1 0 0 0
1 1 0 1
0 1 1 1
0 1 1 1
0 1 1 1
0 1 0 1
0 1 1 1
