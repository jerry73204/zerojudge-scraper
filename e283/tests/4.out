EBADCBBAABFABEAFCDEC
E
C
BADAAFAEADCDEA
BEDFCEDBCEF
AEDCDACFBBE
AAEFEAEACADCFCBCDFA
AFACFBAEBFF
AADAAEAA
DEDCCBEEBB
CEDEBDFBDCEEFDB
ABEBFEAEAEEDDE
D
FE
DACAEFCBCEBDEDBBBAB